Formula/gtk4.rb

```diff
@@ -33,18 +33,25 @@ class Gtk4 < Formula
 
   uses_from_macos "libxslt" => :build # for xsltproc
 
+  on_linux do
+    depends_on "cups"
```

Comment:
```suggestion
  uses_from_macos "libxslt" => :build # for xsltproc
  uses_from_macos "cups"

  on_linux do
```

Reply:
Does Linux ignore `uses_from_macos` lines?

Reply:
nevermind, found the answer: 

> On Linux this will act as depends_on.

Reply:
macOS ignores `uses_from_macos` lines. Any `uses_from_macos` is purely for Linux (unless it has a `since:` argument).